src/inmanta_ui/config.py

```diff
@@ -16,20 +16,112 @@
+#######################################
+
+web_console_enabled = Option(
+    "web-console",
+    "enabled",
+    True,
+    "[DEPRECATED USE :inmanta.config:option:`web-ui.console_enabled`] " "Whether the server should host the web-console or not",
+    is_bool,
+)
 web_console_path = Option(
     "web-console",
     "path",
     "/usr/share/inmanta/web-console",
+    "[DEPRECATED USE :inmanta.config:option:`web-ui.console_path`] "
     "The path on the local file system where the web-console can be found",
     is_str,
 )
 web_console_json_parser = Option(
     "web-console",
     "json_parser",
     "Native",
+    "[DEPRECATED USE :inmanta.config:option:`web-ui.console_json_parser`] "
+    "Whether the web-console should use the 'Native' or the 'BigInt' JSON Parser. "
+    "'BigInt' is useful when the web-console has to show very large integers (larger than 2^53 - 1).",
+    is_str,
+)
+
+#######################################
+# NEW web-console configuration options
+#######################################
+
+web_ui_console_enabled = Option(
+    "web-ui",
+    "console_enabled",
+    True,
+    "Whether the server should host the web-console or not",
+    is_bool,
+    predecessor_option=web_console_enabled,
+)
+web_ui_console_path = Option(
+    "web-ui",
+    "console_path",
+    "/usr/share/inmanta/web-console",
+    "The path on the local file system where the web-console can be found",
+    is_str,
+    predecessor_option=web_console_path,
+)
+web_ui_console_json_parser = Option(
+    "web-ui",
+    "console_json_parser",
+    "Native",
     "Whether the web-console should use the 'Native' or the 'BigInt' JSON Parser. "
     "'BigInt' is useful when the web-console has to show very large integers (larger than 2^53 - 1).",
     is_str,
+    predecessor_option=web_console_json_parser,
+)
+
+#############################
+# Dashboard
+#############################
+
+
+web_ui_dashboard_enabled = Option(
+    "web-ui",
+    "dashboard_enabled",
+    True,
+    "Determines whether the server should host the dashboard or not",
+    is_bool,
+    predecessor_option=dash_enable,
+)
+
+web_ui_dashboard_path = Option(
+    "web-ui",
+    "dashboard_path",
+    "/usr/share/inmanta/dashboard",
+    "The path on the local file system where the dashboard can be found",
+    is_str,
+    predecessor_option=dash_path,
+)
+
+web_ui_dashboard_realm = Option(
+    "web-ui",
+    "dashboard_realm",
+    "inmanta",
+    "The realm to use for keycloak authentication.",
+    is_str,
+    predecessor_option=dash_realm,
+)
+
+web_ui_dashboard_auth_url = Option(
+    "web-ui",
+    "dashboard_auth_url",
+    None,
+    "The auth url of the keycloak server to use.",
+    is_str,
+    predecessor_option=dash_auth_url,
+)
+web_ui_dashboard_client_id = Option(
+    "web-ui",
+    "dashboard_client_id",
+    None,
+    "The client id configured in keycloak for this application.",
+    is_str,
+    predecessor_option=dash_client_id,
```

Comment:
I don't think this is necessary actually. The dashboard will be removed in ISO6, so it doesn't make much sense to migrate the options to a different section. Do you agree?

Reply:
You can ignore this comment. We have to migrate the authentication-related option. So then it makes sense to migrate the other options as well.